odoo/addons/base/models/ir_model.py

```diff
@@ -1259,6 +1250,43 @@ def unlink(self):
 
         return result
 
+    def _process_ondelete(self):
+        """ Process the 'ondelete' of the given selection values. """
+        self = self.filtered(lambda s: s.field_id.store and not s.env[s.field_id.model]._abstract)
+        for selection in self:
+            Model = self.env[selection.field_id.model]
+            field = Model._fields[selection.field_id.name]
```

Comment:
I find this more readable
```suggestion
        for selection in self:
            Model = self.env[selection.field_id.model]
            field = Model._fields[selection.field_id.name]
            if not field.store or Model._abstract:
                continue
```